src/api/hooks/useFetchNviCandidates.ts

```diff
@@ -1,13 +1,19 @@
 import { useQuery } from '@tanstack/react-query';
 import { useTranslation } from 'react-i18next';
-import { FetchNviCandidatesParams, fetchNviAggregations } from '../searchApi';
+import { FetchNviCandidatesParams, fetchNviCandidates } from '../searchApi';
 
-export const useFetchNviCandidates = (queryParams: FetchNviCandidatesParams) => {
+interface FetchNviCandidatesProps {
+  params: FetchNviCandidatesParams;
+  enabled?: boolean;
+}
+
+export const useFetchNviCandidates = ({ params, enabled }: FetchNviCandidatesProps) => {
```

Comment:
Foreslår at vi for alle hooks som tar inn data har et objekt som ligner `useQuery`, sånn at vi ser hva som sendes inn til hva

Reply:
Som vi snakket om i møtet bør ikke målet være å gjøre disse hookene mest mulig generisk. useQuery er allerede generisk og ikke så mange linjer kode, så hvis vi kun wrapper en useQuery men lar all konfigurasjonen sendes inn, går gevinsten bort med å lage denne hooken fordi vi uansett må ha veldig mye spesifikk query-kode i komponenten som bruker den. Jeg foreslår derfor at vi heller har mye mer spesifikke hooks, opptil en per useQuery som også inkluderer all konfigurasjonen. Da tar vi all query-logikken ut av komponenten slik at komponenten først og fremst inkluderer view-logikk og ikke query-logikk.